packages/util-crypto/README.md

```diff
@@ -16,6 +16,20 @@ Functions can be imported as follows:
 import { mnemonicGenerate } from '@polkadot/util-crypto';
 ```
 
-## Utilities
+## Example
 
-[Utilities](SUMMARY.md)
+```js
+import { mnemonicGenerate, mnemonicToBip39, naclKeypairFromSeed, encodeAddress } from '@polkadot/util-crypto';
+
+const mnemonic = mnemonicGenerate()
+console.log('mnemonic:', mnemonic)
+
+const seed = Buffer.from(mnemonicToBip39(mnemonic))
```

Comment:
We generally never want to use this one - it is only there for Ethereum-compatibility (i.e. like in Moonbeam). For Substrate/Polkadot chains this is never used. (This loses entropy along the way, hence it is done differently for Substrate)

Reply:
See this for the seed via minisecret - https://github.com/polkadot-js/common/blob/master/packages/keyring/src/keyring.ts#L185-L187

Reply:
I see, so this is used as seed instead: https://github.com/polkadot-js/common/blob/master/packages/util-crypto/src/mnemonic/toMiniSecret.ts

Reply:
I can't see substantial differences compared to https://github.com/bitcoinjs/bip39/blob/master/src/index.js#L55
Does it just include a default salt/password? https://github.com/polkadot-js/common/blob/master/packages/util-crypto/src/mnemonic/toMiniSecret.ts#L18

Reply:
The big difference is that the minisecret variant goes to entropy (which the older version doesn't) and then uses the rounds with the optional password. So in pure JS (assuming no WASM), -

- substrate version uses mnemonicToEntropy from bip39 and then applies hashing
- the Ethereum version uses mnemonicToSeed from bip39

Reply:
is there any drawback in keeping using old seed generation?

Reply:
It is not compatible with anything in the ecosystem, none of the wallets (apps, Lunie, Enzyme, Math, ...), none of the tools (e.g. subkey) supports the legacy version. But apart from that, cannot comment on the implications, it is good for other chains, so shouldn't have a security mpact.